community-operators/rook/rook.v0.1.0.csv.k8s.yaml

```diff
@@ -0,0 +1,306 @@
+    owned:
+    - kind: CephCluster
+      name: cephclusters.ceph.rook.io
+      version: v1
+      displayName: Ceph Cluster
+      description: Represents a Ceph cluster.
+    - kind: CephBlockPool
+      name: cephblockpools.ceph.rook.io
+      version: v1
+      displayName: Ceph Block Pool
+      description: Represents a Ceph Block Pool.
+    - kind: CephFilesystem
+      name: cephfilesystems.ceph.rook.io
+      version: v1
+      displayName: Ceph Filesystem
+      description: Represents a Ceph Filesystem.
+    - kind: CephNFS
+      name: cephnfses.ceph.rook.io
+      version: v1
+      displayName: Ceph NFS
+      description: Represents a Ceph NFS interface.
+    - kind: CephObjectStore
+      name: cephobjectstores.ceph.rook.io
+      version: v1
+      displayName: Ceph Object Store
+      description: Represents a Ceph Object Store.
+    - kind: CephObjectStoreUser
+      name: cephobjectstoreusers.ceph.rook.io
+      version: v1
+      displayName: Ceph Object Store User
+      description: Represents a Ceph Object Store User.
+    - kind: Volume
+      name: volumes.rook.io
+      version: v1alpha2
+      displayName: Ceph Volume
+      description: Represents a Ceph volume.
+  installModes:
+  - type: OwnNamespace
+    supported: true
+  - type: SingleNamespace
+    supported: true
+  - type: MultiNamespace
+    supported: false
+  - type: AllNamespaces
+    supported: true
+  install:
+    spec:
+      deployments:
+      - name: rook-ceph-operator
+        spec:
+          replicas: 1
+          selector: null # Not supported?
+          strategy: {}
+          template:
+            metadata:
+              creationTimestamp: null
+              labels:
+                app: rook-ceph-operator
+            spec:
+              containers:
+              - args:
+                - ceph
+                - operator
+                env:
+                - name: ROOK_ALLOW_MULTIPLE_FILESYSTEMS
+                  value: "false"
+                - name: ROOK_LOG_LEVEL
+                  value: INFO
+                - name: ROOK_MON_HEALTHCHECK_INTERVAL
+                  value: 45s
+                - name: ROOK_MON_OUT_TIMEOUT
+                  value: 300s
+                - name: ROOK_DISCOVER_DEVICES_INTERVAL
+                  value: 60m
+                - name: ROOK_HOSTPATH_REQUIRES_PRIVILEGED
+                  value: "false"
+                - name: ROOK_ENABLE_SELINUX_RELABELING
+                  value: "true"
+                - name: ROOK_ENABLE_FSGROUP
+                  value: "true"
+                - name: NODE_NAME
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: spec.nodeName
+                - name: POD_NAME
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.name
+                - name: POD_NAMESPACE
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.namespace
```

Comment:
Is this the namespace that the operator watches? In order to support the AllNamespaces installMode, this should be set to `metadata.annotations['olm.targetNamespaces']`.
In the case of an AllNamespace installMode, OLM sets this field to an empty string and this operator would be responsible for watching the cluster for CRs.